playground-next13-rsc/components/Teaser.tsx

```diff
@@ -0,0 +1,8 @@
+"use client";
```

Comment:
Is it for the example ? Maybe you don't need it

Reply:
Not sure I'm following 🤔

Reply:
Do you need a client component here ? You did not use any `useEffect`or `useState`.
If not you could delete `use-client`